### ISSUE TITLE
vsh: cannot use constants from auto-imported modules

### ISSUE THREAD

**Issue by serkonda7** [vlang/v#6372]
**V version:** V 0.1.29 01b28ef
**OS:** Manjaro

**What did you do?**
```v
// script.vsh
println(path_separator)
```

**What did you expect to see?**
Output of `/`
 
**What did you see instead?**
```
[...] error: undefined ident: `path_separator`
[...]
```

**Comment by Henrixounez** [comment]
You can use it, but have to explicitely say it is from the os module as : `os.path_separator`.
Do you wish to be able to use the variable without the `os.` part ?

**Comment by serkonda7** [comment]
Yes, I do not need to type `os. ` before functions, so I don't expect to need it for constants too.

**Comment by danieldaeschle** [comment]
@spytheman how should this behave? 🤔

**Comment by martinskou** [comment]
The auto-import of os seems nice, but is it really?
Audience for .vsh scripts probertly also code normal v and are used to import, so why not just do that?
I suggest do keep the auto os import as is for compatibility, but alter vsh description to discourage the practice of use autoimport.
